README.md

```diff
@@ -8,69 +8,69 @@ XSS/CSRF safe JWT auth designed for SPA
+        - [Token store](#token-store)
+        - [JWT signature](#jwt-signature)
+        - [Request headers and cookies names](#request-headers-and-cookies-names)
+        - [Expiration time](#expiration-time)
+      - [CSRF and cookies](#csrf-and-cookies)
+        - [Refresh with access token](#refresh-with-access-token)
+      - [Refresh token hijack protection](#refresh-token-hijack-protection)
+  - [Flush Sessions](#flush-sessions)
+        - [Sessions namespace](#sessions-namespace)
+        - [Logout](#logout)
+  - [Examples](#examples)
+  - [Contributing](#contributing)
+  - [License](#license)
 
 ## Synopsis
 
-Main goal of this gem is to provide configurable, manageable, and safe stateful sessions based on JSON Web Tokens.
+The primary goal of this gem is to provide configurable, manageable, and safe stateful sessions based on JSON Web Tokens.
 
-The gem stores JWT based sessions on the backend (currently, redis and memory stores are supported), making it possible to manage sessions, reset passwords, logout users in a reliable and secure way.
+The gem stores JWT based sessions on the backend (currently, Redis and memory stores are supported), making it possible to manage sessions, reset passwords and logout users in a reliable and secure way.
 
-It's designed to be framework agnostic yet is easily integrable, and Rails integration is available out of the box.
+It is designed to be framework agnostic, yet easily integrable, and Rails integration is available out of the box.
 
-Core concept behind `jwt_sessions` is that each session is represented by a pair of tokens: access and refresh, and a session store is used to handle CSRF checks and refresh token hijacking. Both tokens have configurable expiration times, but in general refresh token is supposed to have a longer lifespan than an access token. Access token is used to retrieve secured resources and refresh token is used to renew the access token once it's expired. Default token store is based on redis.
+The core concept behind `jwt_sessions` is that each session is represented by a pair of tokens: `access` and `refresh`. The session store is used to handle CSRF checks and prevent refresh token hijacking. Both tokens have configurable expiration times but in general the refresh token is supposed to have a longer lifespan than the access token. The access token is used to retrieve secure resources and the refresh token is used to renew the access token once it has expired. The default token store uses Redis.
 
-All tokens are encoded and decoded by [ruby-jwt](https://github.com/jwt/ruby-jwt) gem, and its reserved claim names are supported as well as it's allowed to configure claim checks and cryptographic signing algorithms supported by it.
+All tokens are encoded and decoded by [ruby-jwt](https://github.com/jwt/ruby-jwt) gem. Its reserved claim names are supported and it can configure claim checks and cryptographic signing algorithms supported by it.
```

Comment:
I found this sentence difficult to understand. I am not an expert in claim checks and cryptographic signing so it was hard to clarify the meaning.